app/src/main/java/org/wikipedia/activity/BaseActivity.kt

```diff
@@ -188,18 +188,13 @@ abstract class BaseActivity : AppCompatActivity(), ConnectionStateMonitor.Callba
     override fun onOptionsItemSelected(item: MenuItem): Boolean {
         return when (item.itemId) {
             android.R.id.home -> {
-                onBackPressed()
+                onBackPressedDispatcher.onBackPressed()
                 true
             }
             else -> false
         }
     }
 
-    override fun onBackPressed() {
-        super.onBackPressed()
-        BreadCrumbLogEvent.logBackPress(this)
```

Comment:
The `BreadCrumbLogEvent.logBackPress(this)` is missing from the new change, is this expected?